modules_core/com.etendorx.auth/src/main/java/com/etendorx/auth/auth/AuthController.java

```diff
@@ -24,9 +26,12 @@ public class AuthController {
   AuthService authServices;
 
   @PostMapping("/authenticate")
-  public ResponseEntity<JwtResponse> authentication(@RequestBody JwtRequest authRequest) {
+  public ResponseEntity<JwtResponse> authentication(@RequestBody JwtRequest authRequest) throws IOException {
     authServices.validateJwtRequest(authRequest);
     UserModel userModel = authServices.validateCredentials(authRequest.getUsername(), authRequest.getPassword());
+    if (userModel.getId().compareTo("100") != 0) {
+      authServices.validateService(userModel.getETRXRxServicesAccessList().get(0).get("rxServiceId"), authRequest.getSecret());
```

Comment:
always exists a service access list ? if this list is empty when try to do get(0) can generate an exception